src/helpers/BasicPointsAndBadges/ConnectExternalAccountActivity.ts

```diff
@@ -1,18 +1,26 @@
 import MyDataHelps from "@careevolution/mydatahelps-js";
 import type { BasicPointsForBadgesActivity, BasicPointsForBadgesActivityState } from "./Activities";
 
+export type ConnectExternalActivityType = "Provider" | "Health Plan" | "Device Manufacturer";
+
 export interface ConnectExternalAccountActivity extends BasicPointsForBadgesActivity {
     type: "connectExternalAccount";
-    providerCategories: string[];
+    providerCategories: ConnectExternalActivityType[];
 }
 
 interface ConnectExternalAccountActivityState extends BasicPointsForBadgesActivityState {
     providersConnected?: number[];
 }
 
 export async function awardConnectExternalAccountActivityPoints(activity: ConnectExternalAccountActivity, activityState: ConnectExternalAccountActivityState) {
+    //convert to array to protect against new types added in consumers but not yet here
+    const activityProviderCategories = activity.providerCategories as string[]; 
```

Comment:
I think you can just use `account.provider.category as ConnectExternalActivityType` below to avoid the typescript error.  If the value from the server is not one of the currently defined values, it just won't match anything when `includes` gets called.